### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sarettasarda**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,23 @@
-# project-smars
\ No newline at end of file
+# Project S-MARS
```
Comment (didmario) [inline README.md]: I like this!!

File: README.md [hunk 2]
```diff
@@ -1 +1,23 @@
...
+Project S-MARS is an home automation system implemented with Arduino and controlled by an Android application. The project is developed by Mario Di Dio, PhD and Sara Craba. Special thanks to Ben and Lynne.
```
Comment (didmario) [inline README.md]: Special thanks to Ben Jirasut and Lynne Freeman

File: README.md [hunk 3]
```diff
@@ -1 +1,23 @@
...
+
+<p>
+The project started after a couple of months that we bought a Etekcity Zap remote control and outlet set for our appartment. Was a cool set, but we constantly lose the remotes (tipically under the couch).<br>
```
Comment (didmario) [inline README.md]: from when we bought.

Comment (didmario) [inline README.md]: It was a cool setup but  we were constantly loosing the remotes

File: README.md [hunk 4]
```diff
@@ -1 +1,23 @@
...
+In a couple of days we solder the remote to an Arduino Yun, creating our first prototype.
```
Comment (didmario) [inline README.md]: We opened up the remote and we understood what needed to be soldered to what and we connected a battery of relay to the remote on one side and to Arduino Yun on the other side. That's how our first prototype was born.

File: README.md [hunk 5]
```diff
@@ -1 +1,23 @@
...
+<p align="center">
+  <img src="https://cloud.githubusercontent.com/assets/5208638/21297181/19d42a96-c530-11e6-830b-ffa5c2745c23.jpg" width="350"/>
+</p>
+
+After some more soldering and a little clean up, we had the final version!
```
Comment (didmario) [inline README.md]: After some more soldering (thanks Ben) and more clean up, we assembled our prototype in a nice clear top box with switches and a single 12V DC power source.

File: README.md (hunk 4 above)
Comment (didmario) [inline README.md]: Of course there are already other commercial available platforms with similar idea but would you buy some bread at the bakery if you are able to do it on your own??

Comment (sarettasarda) [inline README.md]: "Of course there are already other commercial available platforms with similar idea but would you buy some bread at the bakery if you are able to do it on your own??" non mi piace

**Comment by sarettasarda** [comment]
@didmario PR modified.

File: README.md [hunk 6]
```diff
@@ -1 +1,22 @@
-# project-smars
\ No newline at end of file
+# Project S-MARS
+Project S-MARS is an home automation system implemented on Arduino microcontroller and controlled by an Android application. The project is developed by Mario Di Dio, PhD and Sara Craba. Special thanks to Ben Jirasut and Lynne Freeman.
+
+<p>
+The project started after a couple of months from when we bought a Etekcity Zap remote control and outlet set for our appartment. It was a cool setup but we were constantly loosing the remotes (tipically under the couch).<br>
+We opened up the remote and we understood what needed to be soldered to what and we connected a battery of relay to the remote on one side and to Arduino Yun on the other side. That's how our first prototype was born.
```
Comment (didmario) [inline README.md]: a battery of relays

**Comment by sarettasarda** [comment]
Merge Pull Request if ok :)

**Pull request merged**